insights/specs/__init__.py

```diff
@@ -153,7 +153,7 @@ class Specs(SpecSet):
     etc_journald_conf_d = RegistryPoint(multi_output=True)
     etc_journald_conf = RegistryPoint()
     etc_machine_id = RegistryPoint()
-    etc_udev_40_redhat_rules = RegistryPoint(filterable=True)
+    udev_40_redhat_rules = RegistryPoint(filterable=True)
```

Comment:
@akshay196 this breaks backward compatibility for any applications/rules that use this spec.  It would be better not to change this name.

Reply:
@bfahr The spec name `etc_udev_40_redhat_rules` might confuse users if we add new paths.

How about we keep the both the specs and later deprecate `etc_udev_40_redhat_rules` similar to the parser deprecation? Is it possible?